ts/features/payments/details/components/PaymentsMethodDetailsBaseScreenComponent.tsx

```diff
@@ -31,33 +35,61 @@ const PaymentsMethodDetailsBaseScreenComponent = ({
   children
 }: React.PropsWithChildren<Props>) => {
   const isDSenabled = useIOSelector(isDesignSystemEnabledSelector);
+  const insets = useSafeAreaInsets();
+  const translationY = useSharedValue(0);
+  const [titleHeight, setTitleHeight] = React.useState(0);
   const blueHeaderColor = isDSenabled ? IOColors["blueIO-600"] : IOColors.blue;
```

Comment:
It's not related to this PR, but what about handling the "blue" color the new way by using `IOColors[theme["appBackground-accent"]]` or `useAppBackgroundAccentColorName`?

Reply:
The color you indicate by using the hook that returns the "appBackground-accent" seems to be different from the one specified in the [designed figma](https://www.figma.com/design/f7jOjE3RokprMfb7c9NCVQ/Dettaglio-metodo-di-pagamento?node-id=1722-2695&t=X8hi5vfDblZSqHGX-4), what do you think?

| `useAppBackgroundAccentColorName()`|Designed figma|
|-|-|
|<img src="https://github.com/user-attachments/assets/7ca9b072-9950-4c24-a3c8-9bfd0d4835d2" width="300" />|<img src="https://github.com/user-attachments/assets/aa3ea370-3712-49ea-9a85-e11012b46b7a" width="300" />

Reply:
You are right! Your blue is `"interactiveElem-pressed"` 👍

Reply:
If you use `IOColors[theme["interactiveElem-pressed"]]` you'll be compliant to dark mode 🌗 when enabled. What do you think?

Reply:
@shadowsheep1 As @Hantex9 pointed out, the background color is specific to this screen. We should name it `heroBackground-card` or something like that. Let's do that when we edit the whole codebase to unlock the dark mode.

Reply:
Okay, perfect! Good to go then 🚀